modules/contact-us/backend/modules/contact-us/viewsets.py

```diff
@@ -0,0 +1,30 @@
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from sendgrid.helpers.mail import Mail
+from django.conf import settings
+from sendgrid import SendGridAPIClient
+
+
+class ContactUs(APIView):
+	def post(self, request, *args, **kwargs):
+		email = request.data.get('email')
+		name = request.data.get('name')
+		message = request.data.get('message')
+
+		if email and email != '':
+			try:
+				if email:
+					message = Mail(
+						from_email=email,
+						to_emails=settings.TO_EMAILS,
+						subject='Contact Us',
+						html_content='<strong>{}, {}</strong>'.format(name, message))
+					try:
+						sg = SendGridAPIClient(settings.SENDGRID_API_KEY)
+						sg.send(message)
+					except Exception as e:
+						print(e)
```

Comment:
It may be better to write it with a serializer, as it will handle all the validations
```suggestion
                serializer = MessageSerializer(data=request.data)
		if serializer.is_valid(raise_exception=True):
			message = Mail(
					from_email=email,
					to_emails=settings.TO_EMAILS,
					subject='Contact Us',
					html_content='<strong>{}, {}</strong>'.format(name, message))
			try:
				sg = SendGridAPIClient(settings.SENDGRID_API_KEY)
				sg.send(message)
			except Exception as e:
				return Response(
				     {'message': "Service Unavailable, please retry in moment",
				     'status': HTTP_503_SERVICE_UNAVAILABLE},  # <- This shouldn't be needed
				     status=status.HTTP_503_SERVICE_UNAVAILABLE)
		     return Response({'message': "Your query has been sent to admin", 'status': status.HTTP_200_OK}, status=status.HTTP_200_OK)
```